### DISCUSSION TITLE
Недоработка в команде `/giveaway create`

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#23] [Общее]
Если использовать команду `/giveaway create` для создания розыгрыша, но в нём никто не примет участие, то при подведении итогов в сообщении будет отсутствовать строчка из перевода:

![image](https://github.com/LacunaHub/.github/assets/121901513/3eefc460-cf5e-473c-8246-5d1cab44fef1)

Проявляется на всех локализациях

**Comment by danliyev** [comment]
Waiting for the next release
